astropy/samp/hub_script.py

```diff
@@ -106,16 +107,18 @@ def hub_script(timeout=0):
         "specify the output files where redirect the logging messages.",
     )
 
+    log_levels = logging.getLevelNamesMapping().keys()
```

Comment:
Since this is dynamically interpolated into a user visible string, we should take special care that the order levels appear in is stable and expected.
How about this ?
```suggestion
    log_levels = [logging.getLevelName(i) for i in range(50, -1, -10)]
```

Reply:
do you mean order in the sense of level order?
`log_levels = sorted(d:=logging.getLevelNamesMapping(),key=d.get,reverse=True)`

Reply:
yes. I forgot about the `reverse` flag in `sorted`. Your style works for me !

Reply:
done. I do not recall which version of python created `:=
hopefully it's good for astropy, but we can make it more backward-ish

Reply:
it's 3.8. We require >=3.11, so you're safe :)